specification/postgresqlhsc/resource-manager/readme.csharp.md

```diff
@@ -0,0 +1,15 @@
+## C
+
+These settings apply only when `--csharp` is specified on the command line.
+Please also specify `--csharp-sdks-folder=<path to "SDKs" directory of your azure-sdk-for-net clone>`.
+
+```yaml $(csharp)
+csharp:
+  azure-arm: true
+  license-header: MICROSOFT_MIT_NO_VERSION
+  payload-flattening-threshold: 1
+  clear-output-folder: true
+  client-side-validation: false
+  namespace: Microsoft.DBforPostgreSQL
```

Comment:
Namespace should be change to avoid conflict.

Reply:
We will serve the two products from the same namespace and actually RP.

Reply:
If you want to serve it as same namespace, why would you split them in to two spec folder?

Reply:
Because we have very different endpoints and use cases, we don't want to mix them.

Reply:
I confused the namespace with the resource provider name (It's also Microsoft.DBForPostgreSql in resource id). I though we need to change it too.

I changed our namespaces in our SDK readme files.